frame/system/src/lib.rs

```diff
@@ -556,6 +599,11 @@ decl_module! {
 
 		/// Kill the sending account, assuming there are no references outstanding and the composite
 		/// data is equal to its default value.
+		/// 
+		/// # <weight>
+		/// - `O(1)`
+		/// - 1 storage read and deletion.
```

Comment:
It is not this simple, you will need to walk down `fn remove`, where you will see more storage operations and even eventually an event being emitted.

Reply:
missed the `on_killed_account` on the previous analysis :sweat_smile: